### DISCUSSION TITLE
Wifi Problem

### DISCUSSION THREAD

**Discussion by rahidulislam** [linuxdeepin/developer-center#3494] [Troubleshooting | 故障排查]
I was installing linux deepin 23 alpha. but I did not find wifi connection. My wifi works fine with deepin 20.7.1
![image](https://user-images.githubusercontent.com/33887458/202386649-b7e65595-0aea-47e9-86db-8b7010e7a684.png)

**Comment by BLumia** [comment]
Try `sudo lshw -C network` and provide the output. Other wireless connection troubleshooting process on the Internet can also apply to deepin since it's both Linux-based.

btw, v23 is still in alpha stage, so please only use it for testing purpose :)

**Reply by rahidulislam** [reply]
here is my screenshot. 
![image](https://user-images.githubusercontent.com/33887458/202405045-1ca4c4eb-2198-40d3-abc0-cdf0e2458cba.png)
 
![image](https://user-images.githubusercontent.com/33887458/202405128-d4a70067-b6c3-40fe-90ab-c00fdaa49ddc.png)
